action.yml

```diff
@@ -29,27 +32,60 @@ outputs:
 runs:
   using: composite
   steps:
+    - id: registryconfig
+      if: ${{ cloudbees.registries != '' }}
+      name: Resolve source image
+      uses: docker://020229604682.dkr.ecr.us-east-1.amazonaws.com/services/registry-config:0.0.22
```

Comment:
This image is actually available in the public ECR registry, here - `public.ecr.aws/l7o7z1g8/services/registry-config:0.0.22`

Reply:
Updated